docs/adr/0016-publishing-times-and-change-history.md

```diff
@@ -0,0 +1,208 @@
+
+The negatives to this were that there would be no guarantees that this data
+would remain accurate once the work was complete, as Whitehall continues to
+operate; nor would there be an easy way to check whether times are still
+accurate when migrating Whitehall data to Content Publisher. This also didn't
+allow any improvements in removing/editing change history or allowing
+backdating past the first edition.
+
+#### 2. Store times and change history in Content Publisher and sync them to Publishing API
+
+We considered changing Content Publisher to be similar in approach to
+Whitehall and be the canonical source of publishing times and change history.
+Each time Content Publisher content is updated the Publishing API would be
+sent the times and change history.
+
+This approach offers a greater degree of control over data, which allows
+Content Publisher to enhance the backdating feature and provide more support for
+change history removal/editing. This also offers a higher degree of
+compatibility with Whitehall for migrating content due to it being a similar
+model.
+
+This negative is that it increases Content Publisher's responsibilities
+to storing time and change history. It also means that prior to publishing
+content Content Publisher would need to update timestamps in the Publishing API
+which adds an additional API call.
+
+#### 3. Adapt Publishing API to support an altering change history
+
+We considered modifying the Publishing API to allow the altering of change
+histories. This would involve change notes having a particular id value that
+could be used to identify them in requests.
+
+As a positive, this allows each change note to be treated as a separate concept
+to content and, compared to other approaches, allows a reduction in duplicate
+data shared between different editions of content.
+
+A negative of this is that it requires additional development and complexity
+for the Publishing API with a potential migration needed for existing content.
+It also presented a number of complications over whether change history would
+be part of a draft/live workflow, since content already has this concern.
+
+## Decision
+
+We decided to take the approach of storing times and change history in Content
+Publisher.
+
+We felt that this was the most pragmatic approach as it didn't require
+additional development to the Publishing API and was the most compatible with
+Whitehall's approach to change history. The increased control this offered
+was also consistent with a potential future feature of allowing change history
+to be modified via a user interface.
+
+We concluded that the recommended approach for change history in the Publishing
+API was ultimately flawed and should be reconsidered as a recommendation. Our
+concerns were:
+
+- all publishing applications, with the exception of Specialist Publisher,
+  sync their data to Publishing API with little regard to maintaining a
```

Comment:
What do we mean by 'little regard' here?

Reply:
I've made an amend. 

To provide more context: no apps attempt to achieve a parity in what the Publishing API stores in terms of number of editions with what publishers have actually created with all using various means of republishing to achieve syncs.